cvat/apps/dataset_manager/bindings.py

```diff
@@ -602,6 +602,8 @@ def match_dm_item(item, task_data, root_hint=None):
     frame_number = None
     if frame_number is None and item.has_image:
         frame_number = task_data.match_frame(item.image.path, root_hint)
+    if frame_number is None and item.has_image:
```

Comment:
Do we need the "if" on 603 -604 line?

Reply:
Technically - no, it should never match, because `item.image.path` is absolute, and `task_data` has relative paths. This code probably is about to change soon to use only `item.id` with extension included.